apps/spruce/src/pages/adminSettings/tabs/GeneralTab/AuthenticationTab/schemaFields.ts

```diff
@@ -0,0 +1,242 @@
+import { css } from "@emotion/react";
+import widgets from "components/SpruceForm/Widgets";
+import { PreferredAuthType } from "gql/generated/types";
+import {
+  gridWrapCss,
+  fullWidthCss,
+  nestedObjectGridCss,
+} from "../../sharedStyles";
+
+export const radioCSS = css`
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  margin-bottom: 15px;
+  max-width: 100%;
+
+  > div:first-child label {
+    font-weight: 700;
+  }
+`;
+
+export const usersGridCss = css`
```

Comment:
[super nit] It's a little tough to distinguish between users, could we put a border between them?